### DISCUSSION TITLE
Couple of issues with  update to 2.10

### DISCUSSION THREAD

**Discussion by mhk1058** [EdgeTX/edgetx#5001] [Q&A] [answered]
Highlights of 2.10 says New "Eddie the Edgehog" default splash screen - on color display radios also shows the current version of EdgeTX installed. Just updated successfully (inc SD Card) but still have the old splashscreen and no version number on startup. Did I miss something?

Also, volume of welcome message is briefly high on start up then drops to set volume. Previously it behaved as per set volume.

**Comment by pfeerick** [comment, marked as answer]
> default splash screen - on color display radios also shows the current version of EdgeTX installed. Just updated successfully (inc SD Card) but still have the old splashscreen and no version number on startup. 

If when you check in Radio Settings -> Version it says you are running 2.10 and not 2.9 or earlier, you must have a custom splash screen screen (i.e. there is a `/IMAGES/splash.png` file). 

Not sure about what could be causing the volume level to change... given there is also an option to inhibit the startup sound being played, it should have also read the volume level configured and then honor that, maybe something is being read out of sequence.

**Reply by mhk1058** [reply]
Thanks Peter. I just had the default original splashscreen but thanks to your reply I now realise it's a custom image in the images folder - hadn't noticed that before!

Yeh, the sound thing is a minor issue, as you say it must be loading the message ahead of the volume setting but at least I can switch off the welcome message now... could get a bit annoying for the rest of the family if I'm messing around with the radio in the evenings!

**Comment by Sotm12** [comment]
I have edgetx v 2.9.2 on a Radiomaster TX16s and it will NOT update to vers. 2.10.1  Tried Flash Bootloader and Companion. Both say flashed and complete, BUT the vers listed in the system is still 2.9.2 Providence!!!!!!!!!!

**Reply by E120103016** [reply]
https://manual.edgetx.org/installing-and-updating-edgetx/having-trouble-updating#resetting-option-bits

**Comment by pfeerick** [comment]
If you did "flash bootloader", you only updated the bootloader, not the
firmware. Next step is to use the bootloader to update the firmware.

When you used Companion, was this is the radio off when you connected it to
the computer? And did it take over a minute to flash the firmware?
Otherwise you may have done something else, like syncing the SD card after
downloading the  firmware to your computer?

**Reply by Sotm12** [reply]
Thanks for the quick reply. I just got the nightly version update from Github and that update worked. The previous one would NOT. And yes I know how to put the radio in DFU mode.

**Comment by Sotm12** [comment]
It was flashed with Bootloader sorry left out the “with”.
